### ISSUE TITLE
MSI.com - Required user action blocked again (Fanboy's Annoyance List + Fanboy's Cookie List)

### ISSUE THREAD

**Issue by THEtomaso** [ryanbr/fanboy-adblock#482]
### Affected site:
`https://www.msi.com/`

### Example:
`https://www.msi.com/Motherboard/support/Z170A-PC-MATE.html#down-bios`

### Issue:
Required user action (cookie consent) is blocked, preventing downloading of files.

### Problem filters:
* Fanboy's Annoyance List
* Fanboy's Cookie List

### Problem rule:
`###ccc-notify`

### Possible fix:
`msi.com#@##ccc-notify`

--

- OS/version: Windows 8.1 Pro (x64)
- Browser/version: Pale Moon v27.9.3 (x64)
- Adblock Extension/version: uBlock Origin v1.16.4.2 (XUL)

### Reference (previous issue):
https://github.com/ryanbr/fanboy-adblock/issues/475/

**Comment by ryanbr** [comment]
I clicked on the red down arrow, came up with a popup (which showed), pressed okay, and then it downloaded a file. Seems okay?

**Comment by ryanbr** [comment]
if you block any of these;

```
###ccMedal
##.modal-lg
##.modal-dialog
##.modal
##.modal-content
```

That'll prevent the download popup from showing.

**Comment by THEtomaso** [comment]
> Seems okay?

No, this doesn't work like users expect it to.
After clicking on the "ACCEPT" button, nothing happens..
Only after clicking on the download button for a second time, does the download initialize properly.

If you add the exception that i proposed, the general cookie consent for MSI.com will appear, and the download buttons will work like expected afterwards (on the first try).

**Comment by ryanbr** [comment]
Unless you're got other filters, its not affected; see https://github.com/AdguardTeam/AdguardFilters/issues/19361

`@@||civiccomputing.com^*/cookieControl-$script,domain=msi.com`

Currently in the whitelist.

**Comment by THEtomaso** [comment]
This is the general cookie consent popup dialog at MSI.com, which is blocked by `###ccc-notify`:

![filterissue-msi1](https://user-images.githubusercontent.com/9382674/42729296-c4f35e1e-87c2-11e8-8644-319068645a25.jpg)

If the dialog in the screenshot above is blocked, this will appear instead (after clicking on a download button):

![filterissue-msi2](https://user-images.githubusercontent.com/9382674/42729297-cd7a6b9a-87c2-11e8-9ef9-d811ac04f730.jpg)

The `msi.com#@##ccc-notify` exception will ensure that the dialog shown in the first screenshot will appear as soon as users visit the site.
As a result, the dialog show in the second screenshot (and its erratic behaviour) is avoided!

**Comment by ryanbr** [comment]
If you click the download arrow again, does it show? shows for me..

**Comment by THEtomaso** [comment]
Yes, it shows on the second try, but the page doesn't finnish loading and seems to hang on something.
When accepting the general consent dialog instead, everything works like expected.

**Comment by ryanbr** [comment]
it'll be judgement call here, one extra click vs cookie overlay element. I'm happy to leave it
